notary/notary.spec

```diff
@@ -0,0 +1,310 @@
+Requires:      golang(google.golang.org/grpc/credentials)
+
+Provides:      golang(%{import_path}) = %{version}-%{release}
+Provides:      golang(%{import_path}/certs) = %{version}-%{release}
+Provides:      golang(%{import_path}/client) = %{version}-%{release}
+Provides:      golang(%{import_path}/client/changelist) = %{version}-%{release}
+Provides:      golang(%{import_path}/cryptoservice) = %{version}-%{release}
+Provides:      golang(%{import_path}/passphrase) = %{version}-%{release}
+Provides:      golang(%{import_path}/proto) = %{version}-%{release}
+Provides:      golang(%{import_path}/server) = %{version}-%{release}
+Provides:      golang(%{import_path}/server/errors) = %{version}-%{release}
+Provides:      golang(%{import_path}/server/handlers) = %{version}-%{release}
+Provides:      golang(%{import_path}/server/snapshot) = %{version}-%{release}
+Provides:      golang(%{import_path}/server/storage) = %{version}-%{release}
+Provides:      golang(%{import_path}/server/timestamp) = %{version}-%{release}
+Provides:      golang(%{import_path}/signer) = %{version}-%{release}
+Provides:      golang(%{import_path}/signer/api) = %{version}-%{release}
+Provides:      golang(%{import_path}/signer/client) = %{version}-%{release}
+Provides:      golang(%{import_path}/signer/keydbstore) = %{version}-%{release}
+Provides:      golang(%{import_path}/signer/keys) = %{version}-%{release}
+Provides:      golang(%{import_path}/trustmanager) = %{version}-%{release}
+Provides:      golang(%{import_path}/trustmanager/yubikey) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/client) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/data) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/encrypted) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/keys) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/signed) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/store) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/testutils) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/utils) = %{version}-%{release}
+Provides:      golang(%{import_path}/tuf/validation) = %{version}-%{release}
+Provides:      golang(%{import_path}/utils) = %{version}-%{release}
+Provides:      golang(%{import_path}/version) = %{version}-%{release}
+
+%description devel
+%{summary}
+
+This package contains library source intended for
+building other packages which use import path with
+%{import_path} prefix.
+%endif
+
+%if 0%{?with_unit_test} && 0%{?with_devel}
+%package unit-test-devel
+Summary:         Unit tests for %{name} package
+BuildArch:       noarch
+
+%if 0%{?with_check}
+#Here comes all BuildRequires: PACKAGE the unit tests
+#in %%check section need for running
+%endif
+
+# test subpackage tests code from devel subpackage
+Requires:        %{name}-devel = %{version}-%{release}
+
```

Comment:
I have kept the `BuildRequires:` in a single block at the top level instead. (They are not specific to subpackages.)

Added the `Requires:`.

(FWIW I am not certain why packaging the unit tests is supposed to be useful. They are not being developed as an API to be imported by anyone else.)